client/lib/create-config/index.js

```diff
@@ -58,10 +58,9 @@ const config = ( data ) => ( key ) => {
 /**
  * Checks whether a specific feature is enabled.
  *
- * @param {string} feature Feature name
  * @param {object} data the json environment configuration to use for getting config values
+ * curried param {string} feature Feature name
  * @returns {boolean} True when feature is enabled.
- * @api public
```

Comment:
Any info on why this is good to be removed (apart from that it raised errors)?

Reply:
Hmm, no, just trying to get rid of the error. I can re-add it without the `@`, which fixes the error but keeps the info? e88ef3c6bacd404

Reply:
I posted a question here p1605896174269200-slack-C7YPUHBB2

Reply:
Ok, I moved to `@public`, which is the same as `@access public`.  Seems to convey the same information as `@api public` while passing eslint and meeting the JSdoc spec.